src/main/java/org/broadinstitute/hellbender/engine/spark/GATKRegistrator.java

```diff
@@ -30,6 +32,10 @@ public void registerClasses(Kryo kryo) {
 
         // JsonSerializer is needed for the Google Genomics classes like Read and Reference.
         kryo.register(Read.class, new JsonSerializer<Read>());
+
+        kryo.register(Arrays.asList("some","values").getClass(), new JavaSerializer());
+        kryo.register(Collections.nCopies(2,"").getClass(), new JavaSerializer());
```

Comment:
for the nCopies one, would it make sense to use something like https://www.codatlas.com/github.com/apache/giraph/HEAD/giraph-core/src/main/java/org/apache/giraph/writable/kryo/serializers/CollectionsNCopiesSerializer.java or is it too much hassle?